_content/tour/ind/pointers.article

```diff
@@ -183,61 +183,58 @@ No allocation.
         return &u
     }
 
-The escapeToHeap function is using pointer semantics to return a user value back
-to the caller. In other words, the caller gets shared access (an address) to the
-user value being constructed.
+Fungsi escapeToHeap menggunakan semantik nilai bertipe pointer untuk mengembalikan nilai user kepada pemanggil.
+Dengan kata lain, pemanggil mendapatkan akses bersama (sebuah alamat) ke nilai pengguna yang sedang dibuat.
 
-When the escapeToHeap function is called and returns, the user value it constructs
-does still need to exist, since the caller is getting shared access to the value.
-Therefore, the construction of the user value inside of escapeToHeap can’t happen
-on the stack, it must happen on the heap. Yes allocation.
+Ketika fungsi escapeToHeap dipanggil dan mengembalikan nilai, nilai pengguna yang dibuat olehnya
+memang masih perlu ada, karena pemanggil mendapatkan akses bersama ke nilai tersebut.
+Oleh karena itu, konstruksi nilai user di dalam escapeToHeap tidak dapat terjadi di dalam stack,
+melainkan harus terjadi di dalam heap. Ini mengakibatkan alokasi.
 
-Think about what would happen if the user value in the last example was constructed
-on the stack when using pointer semantics on the return.
+Pikirkan apa yang akan terjadi jika nilai user dalam contoh terakhir dibangun di dalam stack
+ketika menggunakan semantik pointer pada pengembalian.
 
 .image /tour/eng/static/img/p2.png
```

Comment:
for the sake of consistency, this reference should point to `ind` instead of `eng` one, forgot that one too, will fix in my pr

Reply:
ack